packages/mobile/components/StrongholdUnlock.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="typescript">
+    import { Button, PasswordInput, Text, HTMLButtonType } from 'shared/components'
+    import { localize } from '@core/i18n'
+    import { unlockStronghold } from '@core/profile'
+
+    export let subtitle: string = ''
+    export let busyMessage: string = ''
+
+    export let onUnlock: (..._: any[]) => unknown = () => {}
+    export let onCancel: () => unknown = () => {}
+
+    let password: string
+    let error = ''
+    let isBusy = false
+
+    async function handleSubmit(): Promise<void> {
+        try {
+            isBusy = true
+            await unlockStronghold(password)
+            onUnlock && onUnlock()
+        } catch (err) {
+            console.error(err)
+            error = localize(err?.message ?? err)
+            isBusy = false
+        }
+    }
+
+    function handleCancelClick(): void {
+        onCancel && onCancel()
+    }
+</script>
+
+<div class="mb-5">
+    <Text type="h4">{localize('popups.password.title')}</Text>
+    <Text type="p" secondary>{subtitle ?? localize('popups.password.subtitle')}</Text>
+</div>
+<form id="password-popup-form" class="flex justify-center w-full flex-row flex-wrap" on:submit={handleSubmit}>
+    <PasswordInput
+        {error}
+        classes="w-full mb-5"
+        bind:value={password}
+        showRevealToggle
+        placeholder={localize('general.password')}
+        autofocus
+    />
+    <div class="flex flex-row justify-between w-full space-x-4">
+        <Button outline classes="w-1/2" onClick={handleCancelClick}>{localize('actions.cancel')}</Button>
```

Comment:
There is neither a cancel nor a back button in the figma design, but if we only use one drawer for the flow, the only option would be to close the drawer complete at this point.